syntax = "proto3";
package alert;




import "monitor_sdk/model/monitor/alert_range.proto";

/*
ListAlertRange请求
*/
message ListAlertRangeRequest {
    /*
    告警事件Id
    */
    string key = 1 ;
}

/*
ListAlertRange返回
*/
message ListAlertRangeResponse {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    消息
    */
    string msg = 2 ;
    /*
    页码
    */
    int32 page = 3 ;
    /*
    分页大小
    */
    int32 page_size = 4 ;
    /*
    总数
    */
    int32 total = 5 ;
    /*
    数据列表
    */
    repeated monitor.AlertRange data = 6 ;
}

/*
ListAlertRangeApi返回
*/
message ListAlertRangeResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    alert.ListAlertRangeResponse data = 4 ;
}

